### DISCUSSION TITLE
Need help! a-rtegen prompt for missing configuration file when executing the example program

### DISCUSSION THREAD

**Discussion by decoski** [toppers/users-forum#101] [Q&A] [answered]
Hello everyone:
i 'm using
atk2-sc3_nios2_dev_gcc-20160324 .tar and a-rtegen-1.3.0.tar for test at cygwin. it's said cfg lost,But the file is generated.
Is the missing file required? Is there an example cfg file that can be sent to me for reference？Thanks！

/cygdrive/e/ProjCyg/rrte-test1228/a-rtegen/sample/sc3/HelloAutosarWithTransAck
$ ./configure.sh
configure: Generating Makefile from ../../../../atk2-sc3/sample/Makefile.
Executable file of the configurator (cfg) is not found.
Generated ./HelloAutosar.arxml
Checking input AUTOSAR XMLs...
Generating .\Rte_InternalDataTypes.arxml ...
There are some insufficient configurations for RTE.
Generating AUTOSAR XMLs to complement the insufficient configurations...
Generating .\Rte_GeneratedEcuc.arxml ...
Generation done.
Generated ./Rte_GeneratedEcuc.yaml
Generated ./HelloAutosar.arxml
Checking input AUTOSAR XMLs...
Generating .\Rte_InternalDataTypes.arxml ...
Generating RTE...
Generation done.

**Comment by mitsut** [comment, marked as answer]
@decoski 

Thank you for using TOPPERS/ATK2　.

Have you downloaded the cfg?
You can download the cfg from the following URL (Sorry Japanese only)
https://www.toppers.jp/cfg-download.html

I think that you are using it in a cygwin environment.
So you can get cfg for Windows target.
https://www.toppers.jp/download.cgi/cfg-mingw-static-1_9_6.zip
and Unpacking it.

Then place the cfg.exe file in the following path
atk2-sc3/cfg/cfg/cfg.exe

And retry running configure.sh.

**Reply by decoski** [reply]
Thank you for your answer. This has solved my problem. The error prompt disappears.
